static/app/views/issueDetails/groupSidebar.tsx

```diff
@@ -54,22 +53,15 @@ type Props = WithRouterProps & {
   event?: Event;
 };
 
-type State = {
-  allEnvironmentsGroupData?: Group;
-  currentRelease?: CurrentRelease;
-  error?: boolean;
-};
-
-class BaseGroupSidebar extends Component<Props, State> {
-  state: State = {};
+function GroupSidebar(props: Props) {
+  const {event, api, group, project, organization, location, environments} = props;
```

Comment:
Why not destructure in the params?
```suggestion
function GroupSidebar({event, api, group, project, organization, location, environments}: Props) {
```